src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -226,4 +228,28 @@ LedgerEntryIsValid::checkIsValid(DataEntry const& de, uint32 version) const
     }
     return {};
 }
+
+std::string
+LedgerEntryIsValid::checkIsValid(ClaimableBalanceEntry const& cbe,
+                                 uint32 version) const
+{
+    if (cbe.claimants.empty())
```

Comment:
Shouldn't this check the rest?

The code here would perform additional checks that we don't use invalid predicate type (like relative timebounds).

Reply:
That's a good point. Added validation on the predicates.